dadi/lib/handlers/image.js

```diff
@@ -914,7 +914,9 @@ ImageHandler.prototype.process = function (sharpImage, imageBuffer) {
 
         case 'png':
           outputFn = 'png'
-          if (options.quality >= 70) outputOptions.compressionLevel = 3
+          // Map options.quality inversely to a compression level between 1 and 9
+          let compressionLevel = parseInt((options.quality * -0.09) + 9)
+          outputOptions.compressionLevel = Math.max(Math.min(compressionLevel, 9), 1)
```

Comment:
The [`compressionLevel` parameter](http://sharp.dimens.io/en/stable/api-output/#parameters_4) seems to be between 0 and 9, so for a `quality` value of `100` should we not get a compression level of `0` instead of `1`?

Reply:
Added a note

Reply:
Perfect, thanks dude!